db/migrate/20191007001034_unfucking_ingredients.rb

```diff
@@ -2,21 +2,25 @@ class UnfuckingIngredients < ActiveRecord::Migration[5.2]
   def change
 
       create_table :recipes do |t|
-        t.references :dishes
-        t.references :ingredients
```

Comment:
I messed up. Uncomment these so we don't get incompatible migrations and I don't lose my head.